test/Microsoft.AspNet.Html.Abstractions.Test/project.json

```diff
@@ -7,16 +7,22 @@
     "Microsoft.AspNet.Html.Abstractions": "1.0.0-*",
     "Microsoft.AspNet.Testing": "1.0.0-*",
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "xunit.runner.aspnet": "2.0.0-aspnet-*"
+    "xunit": "2.1.0"
   },
+  "testRunner": "xunit",
   "commands": {
     "test": "xunit.runner.aspnet"
   },
```

Comment:
Not yet. We still don't have the coreclr runner so we still use dnx to run coreclr tests. I will go away (hopefully) soon